lib/bracket_attributes.ex

```diff
@@ -33,11 +33,29 @@ defmodule BracketAttributes do
     |> concat(choice([word(), double_quoted_string()]))
     |> wrap()
 
+  defparsec(
+    :nested_attribute,
+    choice([
+      string("attr")
+      |> wrap(parsec(:maybe_brackets))
+      |> post_traverse({:flip_attr, []})
+      |> wrap(),
```

Comment:
A nested attribute is an attribute found inside bracket.

We need to handle two cases:
1. The nested attribute is an `attr`. In this case we:
    - flip the order `attr(baz)` -> ["baz", :attr]
    - wrap
2. The nested attribute is any other word. In this case we:
    - do not validate that it has content 
    - parse the content string